src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -468,6 +468,25 @@
                         }
                     ]
                 },
+                {
+                    "Id": "CreateNewVpc",
```

Comment:
For all recipe files, do we have to bump the `Version` value on line 4 now that the tool is GA?

Reply:
I bumped the version